src/main/java/racingcar/view/Inputview.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.model.vo.Round;
+
+public class Inputview {
```

Comment:
여기 살짝 오타가 있는것 같습니다!
다른 클래스는 다 camel case를 적용하셨는데 여기만 적용 안하셔서 남깁니다!
```suggestion
public class InputView {
```

Reply:
엇 그러네요! 세심히 봐주셨군요 :)